### DISCUSSION TITLE
Any largeish Pkl programs out there?

### DISCUSSION THREAD

**Discussion by odenix** [apple/pkl#796] [Q&A]
Having access to real-world largeish Pkl programs would be very helpful for improving interpreter performance.
Ideally these programs would take at least several seconds to run.

**Comment by bioball** [comment]
Agree; this would be good to have. Maybe we can generate something that somewhat emulates how Pkl is used in the real world.

Currently, have ways to test performance against real code-bases, but alas, this is all internal.
